packages/flutter/lib/src/painting/image_cache.dart

```diff
@@ -211,6 +211,12 @@ class ImageCache {
         });
       }
       pendingImage.removeListener();
+      // Remove from live images - the cache will not be able to mark
```

Comment:
Should we call it out in the doc comment of this method that evicting a pending image evicts it from both caches?

Reply:
Done - also expanded the docs on clear and clearLiveImages